Misc/stable_abi.toml

```diff
@@ -1630,20 +1630,8 @@
     added = '3.2'
 [function.Py_GetCopyright]
     added = '3.2'
-[function.Py_GetExecPrefix]
-    added = '3.2'
-[function.Py_GetPath]
-    added = '3.2'
 [function.Py_GetPlatform]
     added = '3.2'
-[function.Py_GetPrefix]
-    added = '3.2'
-[function.Py_GetProgramFullPath]
-    added = '3.2'
-[function.Py_GetProgramName]
-    added = '3.2'
-[function.Py_GetPythonHome]
-    added = '3.2'
 [function.Py_GetRecursionLimit]
```

Comment:
They need to stay here. We can hide them from header files (without `Py_LIMITED_API` set) and further discourage their use, but removing them entirely will break code relying on the stable ABI.